scalyr_agent/monitor_utils/k8s.py

```diff
@@ -877,6 +884,80 @@ def process_object(self, k8s, obj, query_options=None):
         raise NotImplementedError("process_object not implemented for _K8sProcessor")
 
 
+class NamespaceInfo():
+
+    @staticmethod
+    def __flatten_dict(dict_to_flatten):
```

Comment:
Usually flattening a dict is recursive, is this only for the md5 calc below?  Probably better to use json.dumps or even just str.

Reply:
It's for the md5 below only. Switching to json.dumps.